modules/exploits/windows/local/tokenmagic.rb

```diff
@@ -0,0 +1,228 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+require 'msf/core/post/windows/powershell'
+
+class MetasploitModule < Msf::Exploit::Local
+  Rank = ExcellentRanking
+
+  include Msf::Post::Windows::Priv
+  include Msf::Exploit::EXE # Needed for generate_payload_dll
+  include Msf::Post::Windows::FileSystem
+  include Msf::Post::Windows::ReflectiveDLLInjection
+  include Msf::Exploit::FileDropper
+  include Msf::Post::File
+  prepend Msf::Exploit::Remote::AutoCheck
+  include Msf::Post::Windows::Powershell
+
+  prepend Msf::Exploit::Remote::AutoCheck
+
+  def initialize(info = {})
+    super(
+      update_info(
+        info,
+        'Name' => 'TokenMagic UAC Bypass',
+        'Description' => %q{
+          This module leverages a UAC bypass in order to spawn process with system level privileges or abuse a DLL
+          hijacking vulnerability in order to gain SYSTEM level privileges. Windows 7 through to Windows 10 1803
+          are affected.
```

Comment:
```suggestion
          This module leverages a UAC bypass (TokenMagic) in order to spawn a process with SYSTEM-level privileges or abuse a DLL
          hijacking vulnerability in order to gain SYSTEM-level privileges. Windows 7 through Windows 10 1803
          are affected.
```

Grammar and capitalization. Reformat as you see fit.